checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakAnnotatedTypeFactory.java

```diff
@@ -110,9 +109,7 @@ public ResourceLeakAnnotatedTypeFactory(BaseTypeChecker checker) {
    * @return true iff the given element is a final field with non-empty @MustCall obligation
    */
   /*package-private*/ boolean isCandidateOwningField(Element element) {
-    return (element.getKind().isField()
-        && ElementUtils.isFinal(element)
-        && !getMustCallValue(element).isEmpty());
+    return (element.getKind().isField() && !getMustCallValue(element).isEmpty());
```

Comment:
This change is inconsistent with the documentation, which says the field must be final for the method to return true.